conda/base/context.py

```diff
@@ -446,6 +444,18 @@ def experimental_solver(self):
         aliases=("conda-build", "conda_build"),
     )
 
+    ####################################################
+    #               Plugin Configuration               #
+    ####################################################
+
+    _no_plugins = ParameterLoader(
+        PrimitiveParameter(NO_PLUGINS, element_type=(bool, NoneType)),
+        aliases=(
+            "no_plugins",
+            "no_plugin",
```

Comment:
I don't think we need `no_plugin` here

Reply:
I was thinking of the most common and harmless typo instance to cover here, but I will go ahead and remove it! 👍

Reply:
> I was thinking of the most common and harmless typo instance to cover here
I think that's a good idea.